src/identity/interaction/email-password/credentials/ListCredentialsHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { BasicRepresentation } from '../../../../http/representation/BasicRepresentation';
+import type { Representation } from '../../../../http/representation/Representation';
+import { APPLICATION_JSON } from '../../../../util/ContentTypes';
+import type { AccountStore } from '../storage/AccountStore';
+import type { CredentialsHandlerInput } from './CredentialsHandler';
+import { CredentialsHandler } from './CredentialsHandler';
+
+/**
+ * Returns a list of all credential tokens associated with this account.
```

Comment:
Wait, we said above that we would never return them again?
We probably want to truncate them to the last 6 letters or so.

Reply:
We will never return the secrets again. To do a client_credentials request you need 2 things: a client_id and a client_secret. These need to be send as a base64 encoding of "client_id:client_secret" to request an access token. What we do is (mis)use the client_id as the API "key". The user can provide a name so if they have a look at the list afterwards they can remember why this key was made. It's also what they need to send to us to delete it again so it's important they can see these names. The second part is a secret that we generate when a token gets created. That secret is only returned as a response of the create request and can never be accessed at a later point by the user.

Reply:
Okay; can yhe comments reflect this?